### PR TITLE
New "columns" display mode

### PR DESCRIPTION
**Pull request by Zenoo**
Setting `config_display: columns` in the action and using  
`Metrics`

in your README will create a two columns display on desktop:  

This PR is a followup of https://github.com/lowlighter/metrics/discussions/368

### CODE REVIEW

File: source/app/web/statics/app.placeholder.js [hunk 1]
```diff
@@ -56,6 +56,7 @@
       animated: false,
       //Display size
       large: set.config.display === "large",
+      columns: set.config.display === "colmuns",
```
Comment (lowlighter) [inline source/app/web/statics/app.placeholder.js:59]:
```suggestion
      columns: set.config.display === "columns",
```

File: README.md [hunk 2]
```diff
@@ -529,6 +529,8 @@ Update your README.md to embed your metrics:
 ![Metrics](https://github.com/my-github-user/my-github-user/blob/master/github-metrics.svg)
 <!-- If you're using "main" as default branch -->
 ![Metrics](https://github.com/my-github-user/my-github-user/blob/main/github-metrics.svg)
+<!-- If you're using the "columns" display mode -->
+<img src="https://github.com/my-github-user/my-github-user/blob/master/github-metrics.svg" alt="Metrics" width="100%">
```
Comment (lowlighter) [inline README.md:533]: Can you patch [`.github/readme/partials/setup/action/setup.md`](https://github.com/lowlighter/metrics/blob/master/.github/readme/partials/setup/action/setup.md) with this change instead? The readme is now auto-generated from these files so changes won't be reflected when the actions bot rebuild it

**Review by lowlighter — COMMENTED** [review]
Looks good 👍 
Thanks a lot for this feature!

File: source/app/web/statics/app.placeholder.js (hunk 1 above)
Comment (Zenoo) [reply source/app/web/statics/app.placeholder.js]: Thanks for checking, the typo should be fixed now

File: README.md (hunk 2 above)
Comment (Zenoo) [reply README.md:533]: Thanks, I didn't know it was rebuilt ! It's done

**Review by lowlighter — APPROVED** [review]

**Pull request merged**